enterprise/frontend/src/embedding-sdk-ee/subscriptions/DashboardSubscriptionsButton.unit.spec.tsx

```diff
@@ -27,19 +71,73 @@ describe("DashboardSubscriptionsButton", () => {
     expect(
       await screen.findByRole("button", { name: "Subscriptions" }),
     ).toBeInTheDocument();
-    const gets = await findRequests("GET");
-    expect(gets).toHaveLength(1);
+    const subscriptionChannelInfoRequests =
+      await getSubscriptionChannelInfoRequests();
+    expect(subscriptionChannelInfoRequests).toHaveLength(1);
   });
 
   it("should not render the subscriptions button when email is not configured", async () => {
     setup({ emailConfigured: false });
     // Ensure the API finishes loading, otherwise, the assertion below will always pass because the API hasn't returned yet
     await waitFor(async () => {
-      const gets = await findRequests("GET");
```

Comment:
get used to be for only `/api/pulse/form_input` now there are a bunch of dashboard calls too.